posts/taking-learning-outside-the-classroom-into-the-cloud.md

```diff
@@ -0,0 +1,54 @@
+image: 
+---
+
+A love for learning is a quality that I have always looked for in peers and teammates, and it’s one that I’ve had the opportunity and privilege to invest in daily over the past 9 months. To me, a commitment to learning requires humility to accept that we often have far more questions than answers to a problem. But when we commit ourselves to learning, we are promising to always seek self-improvement: to honor our questions, seek better understanding, and ultimately be a better version of ourselves today than we were yesterday. In the workplace, a love for learning means we are honest and accountable with our teammates, and more importantly we support and believe in each other’s growth. It was this exact quality that drew me to working with Ippon Technologies, USA.
+
+In fact, it was around this time last year that I accepted an offer to intern over the summer with Ippon in the Richmond, VA office. Thrilled to return to my hometown for the summer, I also marveled at the idea of finally getting hands-on experience in the software industry with talented folks whose values aligned with mine. However, as is the case with the rest of the world, the global pandemic quickly uprooted my plans for the future. Faced with growing uncertainty about the year ahead, I decided to delay my senior year of college, and I was extremely fortunate to extend my remote internship with Ippon into a full-time student software engineer position.
+
+I cannot express enough gratitude to the many mentors who have taught me over the past 9 months, especially my managers Ben Scott and Amine Ouali Alami, whose humility and leadership I can only hope to emulate one day. This blog post chronicles my 9-month deep-dive into the industry and reflects on the challenges I’ve encountered, as well as the lessons I’ve learned.
+
+
+<h1>Interning with Ippon, Summer 2020 </h1>
+
+During my internship with Ippon, I had my first introduction to a number of software engineering concepts and tooling, which, to the average software engineer, are run-of-the-mill household names. I learned about and gained experience with agile development, version control, layered architectures, and continuous integration/continuous development processes. If that wasn’t enough, I was also introduced to numerous technologies such as AWS cloud resources, React, Redux, Spring Boot, PostgreSQL, and most importantly, Ippon's in-house JHipster platform.
+
+In particular, I spent the summer working alongside a team of three other interns developing an in-house application to collect results on clients’ assessments of their design, CI/CD, and testing processes, among other areas of software development workflow. While we each set out working on our own stories in our relative comfort zones (front-end versus back-end), by a few weeks into the project we had all stepped into full-stack development roles and even began designing new components for the application.
```

Comment:
While we each set out working on our own stories in our relative comfort zones (front-end versus back-end), we had all stepped into full-stack development roles and even began designing new components for the application within a few weeks.